### PR TITLE
Test update for size tracing code

### PR DESCRIPTION
**Pull request by apaszke**

### CODE REVIEW

File: test/test_operators.py [hunk 1]
```diff
@@ -391,7 +391,7 @@ def _run_forward(x, scale, shift, eps):
                 t = x.view(x.size(0), x.size(1), n)
                 mean = torch.mean(t, 2).unsqueeze(2).unsqueeze(3).expand_as(x)
                 # Calculate the biased var. torch.var returns unbiased var
-                var = torch.var(t, 2).unsqueeze(2).unsqueeze(3).expand_as(x) * ((n - 1) / float(n))
+                var = torch.var(t, 2).unsqueeze(2).unsqueeze(3).expand_as(x) * ((float(n) - 1) / float(n))
```
Comment (bddppq) [inline test/test_operators.py]: Why is this needed?

Comment (apaszke) [reply test/test_operators.py]: Because `n` will be a PyTorch scalar of type `long`, but will be involved in float math (because you'll multiply it by the output of `torch.var`). It's all perfectly traceable, it's just that we don't support cross-type ops as well as we would like to, and as such this line starts to cause an error.

Comment (bddppq) [reply test/test_operators.py]: Oh I see previously it's all done in the python land so it didn't cause error.

Comment (apaszke) [reply test/test_operators.py]: Yep. Previously `n` was a Python scalar, so we didn't care too much about its type. Now it's a PyTorch scalar, and right now we have those dumb restrictions...
